source/agora/common/Types.d

```diff
@@ -76,4 +76,38 @@ public struct Height
 
     /// Provides implicit conversion to `ulong`
     public alias value this;
+
+    /// Helper type for `toString`
+    private alias SinkT = void delegate(scope const(char)[] v) @safe;
```

Comment:
I don't think we need this, it's only used from within a function

Reply:
I Changed the string conversion method.
Using std.conv